vector/src/main/java/im/vector/app/features/settings/VectorSettingsPinFragment.kt

```diff
@@ -50,7 +50,9 @@ class VectorSettingsPinFragment :
     override var titleRes = R.string.settings_security_application_protection_screen_title
     override val preferenceXmlRes = R.xml.vector_settings_pin
 
-    private val biometricHelper = biometricHelperFactory.create(defaultLockScreenConfiguration.copy(mode = LockScreenMode.CREATE))
+    private val biometricHelper by lazy {
+        biometricHelperFactory.create(defaultLockScreenConfiguration.copy(mode = LockScreenMode.CREATE))
+    }
```

Comment:
it could also be a `lateinit var` and be created in `fun onCreate()`. Not sure if it's better, but at least it will remove the overhead of using `lazy`.

Reply:
(IMO we should use `by lazy` when on object would eventually be not used - which is not the case here)

Reply:
Sorry, when it was approved I enabled automatic merge and by the time I read that message it was already merged. I can open a new PR changing lazy to lateinit and instantiate it in `onCreate` if you think it's worth it.

Reply:
No worries, this is not urgent.